src/protections/ProtectionManager.ts

```diff
@@ -159,7 +160,7 @@ export class ProtectionManager {
         await this.enabledProtectionsManager.start();
         this.mjolnir.reportManager.on("report.new", this.handleReport.bind(this));
         this.mjolnir.matrixEmitter.on("room.event", this.handleEvent.bind(this));
-        for (const protection of PROTECTIONS) {
+        for (const protection of this.protection_classes) {
```

Comment:
Maybe we could just move the list of protections (formerly) `PROTECTIONS` to here? and then pass `_protections` to the EnabledProtectionsManager`?